SignIn/Check.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SignIn
+{
+    class CheckPassword
+    {
+        public bool Password(string password, int min)
+        {
+            int length = password.Length;
+            return  length>= min;
+        }
+        public bool Alphabet(string password)
+        {
+            bool flag = true;
+            for (int i = 0; i < password.Length; i++)
+            {
+                if (!(password[i] >= 'a' && password[i] <= 'z' || password[i] >= 'A' && password[i] <= 'Z' || password[i] >= '0' && password[i] <= '9' || password[i] >= '!' && password[i] <= '/' || password[i] >= ':' && password[i] <= '@' || password[i] >= '[' && password[i] <= '`' || password[i] >= '{' && password[i] <= '~'))
+                {
+                    flag = false;
+                    break;
+                }
+            }
+
+            if (flag == true)
+            {
+                return true;
+            }
+            return false;

```

Comment:
Легко заменяется одной строчкой

```suggestion
            return flag;
```